ironfish-cli/src/commands/miners/mined.ts

```diff
@@ -1,136 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-import {
-  Assert,
-  AsyncUtils,
-  CurrencyUtils,
-  GENESIS_BLOCK_SEQUENCE,
-  Meter,
-  TimeUtils,
-} from '@ironfish/sdk'
-import { CliUx, Flags } from '@oclif/core'
-import readline from 'readline'
-import { parseNumber } from '../../args'
-import { IronfishCommand } from '../../command'
-import { RemoteFlags } from '../../flags'
-import { ProgressBar } from '../../types'
-import { linkText } from '../../utils/terminal'
-
-export class MinedCommand extends IronfishCommand {
```

Comment:
I could update this command to scan the chain instead for now, but I'm not sure whether it's useful for testnet participants now.

Reply:
Discussed and decided that since we're not doing a release in a while, we have time to stop informing people to use this command, or rewrite it with the new wallet.